src/lib/ui/ExtensionsList.svelte

```diff
@@ -1,13 +1,54 @@
 <script lang="ts">
-import type { ExtensionByCategoryInfo } from '$lib/api/extensions-info';
+import type { CatalogExtensionInfo, ExtensionByCategoryInfo } from '$lib/api/extensions-info';
 
+import ExtensionByCategoryCard from './ExtensionByCategoryCard.svelte';
 import ExtensionsByCategory from './ExtensionsByCategory.svelte';
 
-const { extensionsByCategories }: { extensionsByCategories: ExtensionByCategoryInfo[] } = $props();
+export let extensionsByCategories: ExtensionByCategoryInfo[];
```

Comment:
why are you reverting to svelte 4?

Reply:
will fix, bad habit using svelte 4 for everything haha!